tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -295,31 +298,43 @@ def start_dhcp_monitor_debug_counter(duthost):
 
 
 def test_dhcp_relay_default(ptfhost, dut_dhcp_relay_data, validate_dut_routes_exist, testing_config,
-                            toggle_all_simulator_ports_to_rand_selected_tor_m):     # noqa F811
+                            rand_unselected_dut, toggle_all_simulator_ports_to_rand_selected_tor_m):     # noqa F811
     """Test DHCP relay functionality on T0 topology.
        For each DHCP relay agent running on the DuT, verify DHCP packets are relayed properly
     """
+
     testing_mode, duthost, testbed_mode = testing_config
 
     if testing_mode == DUAL_TOR_MODE:
         skip_release(duthost, ["201811", "201911"])
 
-    skip_dhcpmon = (testing_mode == DUAL_TOR_MODE or
-                    any(vers in duthost.os_version for vers in ["201811", "201911", "202111"]))
+    skip_dhcpmon = "201811" in duthost.os_version or "201911" in duthost.os_version
```

Comment:
@kellyyeh What about 202111?

Reply:
It accidentally got overwritten, thanks for catching. Reverted